src/common/classes/vector.h

```diff
@@ -135,7 +157,11 @@ class Vector
 
 protected:
 	FB_SIZE_T count;
-	T data[Capacity];
+	union
+	{
+		T data[Capacity];
+		A align;
+	};
 };
```

Comment:
@asfernandes Instead readme added a chapter to OO API documentation. Usage sample also reworked to cover various batch aspects.